backend/clubs/serializers.py

```diff
@@ -849,6 +850,24 @@ class Meta:
         fields = ("club", "person")
 
 
+class FavoriteTimeSerializer(serializers.ModelSerializer):
+    """
+    Used by clubs to get a list of people who have favorited their club as
+    well as when those people have favorited (for analytics).
+    """
+
+    person = serializers.HiddenField(default=serializers.CurrentUserDefault())
+    club = serializers.SlugRelatedField(queryset=Club.objects.all(), slug_field="code")
+    hour = serializers.SerializerMethodField("get_hour")
```

Comment:
Ideally, you should do this aggregation in SQL, which should be much faster than doing it in Python.